src/main/java/io/kestra/plugin/openai/Responses.java

```diff
@@ -0,0 +1,296 @@
+        ),
+        @Example(
+            full = true,
+            title = "Structured output with web search tool",
+            code = """
+                id: responses_json_search
+                namespace: company.team
+
+                inputs:
+                  - id: prompt
+                    type: STRING
+                    defaults: "List recent trends in workflow orchestration. Return as JSON."
+
+                tasks:
+                  - id: trends
+                    type: io.kestra.plugin.openai.Responses
+                    apiKey: "{{ secret('OPENAI_API_KEY') }}"
+                    model: gpt-4.1
+                    input: "{{ inputs.prompt }}"
+                    text:
+                      format:
+                        type: json_object
+                    toolChoice: required
+                    tools:
+                      - type: web_search_preview
+
+                  - id: log
+                    type: io.kestra.plugin.core.log.Log
+                    message: "{{ outputs.trends.outputText }}"
+            """
+        )
+    }
+)
+public class Responses extends AbstractTask implements RunnableTask<Responses.Output> {
+
+    @Schema(
+        title = "Model name to use (e.g., gpt-4o)",
+        description = "See the [OpenAI model's documentation](https://platform.openai.com/docs/models)"
+    )
+    @NotNull
+    private Property<String> model;
+
+    @Schema(
+        title = "Input for the model"
+    )
+    @NotNull
+    private Property<Object> input;
+
+    @Schema(
+        title = "Text response configuration",
+        description = "Configure the format of the model's text response"
+    )
+    private Property<Map<String, Object>> text;
+
+    @Schema(
+        title = "List of built-in tools to enable",
+        description = "e.g., web_search, file_search, function calling"
+    )
+    private Property<List<Tool>> tools;
+
+    @Schema(
+        title = "Controls which tool is called by the model",
+        description = "none: no tools, auto: model picks, required: must use tools"
+    )
+    private Property<String> toolChoice;
+
+    @Schema(
+        title = "Whether to persist response and chat history",
+        description = "If true (default), persists in OpenAI's storage"
+    )
+    private Property<Boolean> store;
+
+    @Schema(
+        title = "ID of previous response to continue conversation"
+    )
+    private Property<String> previousResponseId;
+
+    @Schema(
+        title = "Reasoning configuration",
+        description = "Configuration for model reasoning process"
+    )
+    private Property<Map<String, String>> reasoning;
+
+    @Schema(
+        title = "Maximum tokens in the response"
+    )
+    private Property<Integer> maxOutputTokens;
+
+    @Schema(
+        title = "Sampling temperature (0-2)"
+    )
+    private Property<Double> temperature;
+
+    @Schema(
+        title = "Nucleus sampling parameter (0-1)"
+    )
+    private Property<Double> topP;
```

Comment:
```suggestion
    private Property<@Max(1) Double> topP;
```

Better to add validation when possible

Reply:
Same for temperature